stacker_blueprints/sns.py

```diff
@@ -53,17 +84,62 @@ def create_template(self):
         for topic_name, topic_config in variables["Topics"].iteritems():
             self.create_topic(topic_name, topic_config)
 
+    def create_sqs_policy(self, topic_name, topic_arn, topic_subs):
+        """
+        This method creates the SQS policy needed for an SNS subscription. It
+        also takes the ARN of the SQS queue and converts it to the URL needed
+        for the subscription, as that takes a URL rather than the ARN.
+        """
+        t = self.template
+
+        arn_endpoints = []
+        url_endpoints = []
+        for sub in topic_subs:
+            arn_endpoints.append(sub["Endpoint"])
+            split_endpoint = sub["Endpoint"].split(":")
+            queue_url = "https://%s.%s.amazonaws.com/%s/%s" % (
```

Comment:
I prefer this because I don't like comments:

```
queue_url = "https://%(sqs)s.%(region)s.amazonaws.com/%(aws_id}s/%(queue_name)s" % {
    "sqs": split_endpoint[2],
    "region": split_endpoint[3],
    "aws_id": split_endpoint[4],
    "queue_name": split_endpoint[5],
}
```